x-pack/plugin/kql/src/main/java/org/elasticsearch/xpack/kql/KqlFeatures.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.kql;
+
+import org.elasticsearch.Build;
+import org.elasticsearch.common.util.set.Sets;
+import org.elasticsearch.features.FeatureSpecification;
+import org.elasticsearch.features.NodeFeature;
+
+import java.util.Collections;
+import java.util.Set;
+
+public class KqlFeatures implements FeatureSpecification {
+
+    public static final NodeFeature KQL_QUERY_SUPPORTED = new NodeFeature("kql_query_supported");
```

Comment:
We don't need a node feature. This should simply be a capability for testing. I don't see any usages that actually requires a node feature.